src/uwtools/tests/drivers/test_chgres_cube.py

```diff
@@ -130,40 +131,18 @@ def test_ChgresCube_run_local(driverobj):
 
 
 def test_ChgresCube_runscript(driverobj):
-    dst = driverobj._rundir / "runscript.chgres_cube"
-    assert not dst.is_file()
-    driverobj.runscript()
-    with open(dst, "r", encoding="utf-8") as f:
-        lines = f.read().split("\n")
-    # Check directives:
-    assert "#SBATCH --account=me" in lines
-    assert "#SBATCH --time=00:02:00" in lines
-    # Check environment commands:
-    assert "cmd1" in lines
-    assert "cmd2" in lines
-    # Check execution:
-    assert "srun --export=ALL --ntasks $SLURM_CPUS_ON_NODE /path/to/chgres_cube" in lines
-    assert "test $? -eq 0 && touch %s/done" % driverobj._rundir
+    with patch.object(driverobj, "_runscript") as runscript:
+        driverobj.runscript()
+        runscript.assert_called_once()
+        args = ("envcmds", "envvars", "execution", "scheduler")
+        types = [list, dict, list, Slurm]
+        assert [type(runscript.call_args.kwargs[x]) for x in args] == types
```

Comment:
This test (and the corresponding ones for other drivers) can be simpler now, as the bulk of the logic in the `Driver` class has its own test.